app/views/search-for-your-gp.nunjucks

```diff
@@ -10,7 +10,7 @@
 {% endif %}
 {% endblock %}
 
-{% block pageTitle %}Find a GP to book an appointment with{% endblock %}
+{% block pageTitle %}Book a GP appointment online{% endblock %}
```

Comment:
Is it worth extracting this out to a template variable to remove the duplication?

e.g.
```
{% set title = "Book a GP appointment online" %}
{% block header %}
<h1 class="page-title">{{title}}</h1>
{% endblock %}
...
{% block pageTitle %}{{title}}{% endblock %}
...
```
Not tested this code but it gives a flavour.